apps/paf/translations/src/en/validation.json

```diff
@@ -213,11 +215,11 @@
   },
   "report-person-occupation-company-phone": {
     "required": "Enter a valid phone number",
-    "internationalPhoneNumber": "Please enter a valid phone number",
+    "internationalPhoneNumber": "Enter a valid phone number",
     "maxlength": "You can't use more than {{maxlength}} characters for your answer"
   },
   "company-phone": {
-    "required": "Enter a valid phone number",
```

Comment:
Is this one not required any more? It might not be, but it looks like a lot of the other phone numbers still have the "required" validation too. No worries if not.

Reply:
Not sure is required but i can include it